airflow/models/serialized_dag.py

```diff
@@ -37,10 +37,25 @@
 from airflow.utils import timezone
 from airflow.utils.session import provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
+from airflow.www.security import AirflowSecurityManager
 
 log = logging.getLogger(__name__)
 
 
+class SimpleSecurityManager(AirflowSecurityManager):
+    """Security Manager that doesn't need the whole flask app"""
+
+    def __init__(self):  # pylint: disable=super-init-not-called
+        self.session = None
+
+    @property
+    def get_session(self):
+        return self.session
+
+
+security_manager = SimpleSecurityManager()
```

Comment:
Ok, after thinking more about this, I don't think we should be extending the security manager into the `/airflow/models` directory. I'd much rather create a `sync-permissions` API endpoint if one doesn't exist, and hitting that from the CLI via a separate HTTP request.